### PR TITLE
Add support for the alex linter

### PR DESCRIPTION
**Pull request by joshblack**
Adds the [alex](https://github.com/get-alex/alex) project to help catch insensitive, inconsiderate writing in our documentation.

#### Changelog

**New**

- A new `yarn lint:docs` command for linting documentation

**Changed**

- The project's ci-check now includes `yarn lint:docs`
- Files have been updated to address the linter violations

**Removed**

### CODE REVIEW

**Comment by joshblack** [comment]
FYI @connor-leech, thought this was a cool project and wanted to see what it'd be like to include it. Curious what your thoughts are 👀

File: docs/guides/accessibility.md [hunk 1]
```diff
@@ -150,11 +152,15 @@ component with the necessary keyboard functionality.
 
 ### Screen Magnifier testing🔎
```
Comment (connor-leech) [inline docs/guides/accessibility.md]: Is this a proper noun?

File: docs/guides/endgame.md [hunk 2]
```diff
@@ -757,7 +758,7 @@ following steps to release:
 ### Release Candidate releases
```
Comment (connor-leech) [inline docs/guides/endgame.md]: Is this a proper noun? I don't think it is.

File: docs/guides/accessibility.md (hunk 1 above)
Comment (joshblack) [reply docs/guides/accessibility.md]:
```suggestion
### Screen magnifier testing🔎
```

File: docs/guides/endgame.md (hunk 2 above)
Comment (joshblack) [reply docs/guides/endgame.md]:
```suggestion
### Release candidate releases
```

**Review by alisonjoseph — APPROVED** [review]
❤️ Love this! Thanks for adding it @joshblack

**Comment by alisonjoseph** [comment]
Would we want to add this to our website and see what happens also? @connor-leech  (does it work on mdx @joshblack ?)

**Comment by connor-leech** [comment]
@alisonjoseph I'd love to see what happens if we ran this on the website.

**Comment by joshblack** [comment]
bump @asudoh @vpicone if you have a sec to review!

File: docs/guides/endgame.md [hunk 3]
```diff
@@ -507,10 +508,10 @@ already exist in the project. These updates are reserved for
 8. Manually check to see that the `latest` tags were updated for the packages
    correctly by going to [`npm`](https://www.npmjs.com), searching the packages,
    viewing the versions and making sure the `latest` version matches the version
-   you just released.
```
Comment (vpicone) [inline docs/guides/endgame.md:512]: hmmm I think this one is more about time proximity than 'its as simple as', although I don't know if removing it substantially changes the sentance.

**Review by vpicone — APPROVED** [review]
Awesome! I heard about this linter recently, thanks for taking the time to integrate it.

**Pull request merged**